docs/user/reference/general/initializing.md

```diff
@@ -113,9 +113,13 @@ class SampleApplication : Application() {
 }
 ```
 
-The Glean Kotlin SDK does not support use across multiple processes, and must only be initialized on the application's main process. Initializing in other processes is a no-op.
+The Glean Kotlin SDK supports use across multiple processes. To configure this you call `initialize` as usual from your main process. From the secondary process you can call `initialize` again with the `dataPath` parameter. This allows you to store data locally in a separate location for the secondary process.
```

Comment:
I think I'd rather the data-path be in the Configuration object with a default to the current `glean_data`. My fear is that users will see this parameter and think that they need to supply the data path for Glean when they shouldn't.